ADA/ada.qmd

```diff
@@ -19,57 +17,194 @@ It brings together:
+### Making the most of Databricks 
 
 ---
 
-## How Databricks works
+In order to take advantage of the benefits listed above, we recommend that you and your team:
 
-See the [ADA support page on how Databricks works](https://analytical-data-access.azurewebsites.net/get-support/how-databricks-works).
+* Arrange data migration and access to Databricks. Onboarding is currently voluntary, so it's a great opportunity to move early and have more time to get familiar with the system and undertake training. Get in touch with [the ADA team](mailto:ADAPTTeam@education.gov.uk) to discuss onboarding with them. 
 
-## Differences between IDEs and Notebooks
+* Take part in Databricks training for R and / or SQL. R training is currently offered by the ADA team on a monthly basis and SQL training is in the process of being developed. Upcoming training is advertised on the [ADA user group Teams channel](https://teams.microsoft.com/l/team/19%3AGQW24K3V6M873QzUwpBIuyf8KZCxYByRawVsKQvpe-81%40thread.tacv2/conversations?groupId=c70bb59d-c5df-4af5-849f-36196ed14ba3&tenantId=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9).
 
-RStudio is an IDE (Integrated Development Environment) whereas Databricks makes use of Notebooks, where you write your code in blocks contained in individual cells. There are advantages and disadvantages to both methods, and [this NHS-R article](https://nhsdigital.github.io/rap-community-of-practice/implementing_RAP/notebooks_versus_ide_development/#what-is-this-guide-for) outlines some of the differences you might come across when switching from using IDEs to Notebooks. 
+* Be proactive in determining whether any changes will be necessary for your existing code and set time aside to make these changes - see the guidance in the [what this means for existing code](/ADA/ada.html#what-this-means-for-existing-code) section below. 
+
+::: {.callout-note}
+The earlier your data is migrated, the more time you will have to dual run code from your existing methodology against code run on the Databricks platform. We encourage all analysts to engage with training and ask the ADA team about any questions or concerns as soon as possible so that you are prepared for any upcoming changes.
+:::
+
+For teams that have invested in RAP principles and already have existing code pipelines, it is possible to take advantage of cloud computing and use of data in the Databricks platform with minimal changes. This is covered in more detail in the [what this means for existing code](/ADA/ada.html#what-this-means-for-existing-code) section below. 
+
+::: {.callout-important}
+Once your team's data has been migrated, you will **not** immediately lose access to your existing data sources, so you will be able to dual run your code using data from SSMS and data from the Databricks SQL warehouse to ensure that everything is working as it should be. Once you have confirmed that you are happy with the data in Databricks, your access to the legacy platform will be switched off. 
```

Comment:
Assume the ADA people have confirmed this is the case too right? 
When we send to them for review can we call out some specific lines, like this one, that are important they still commit to?

Reply:
Yeah I checked in all their info on the intranet and they mention it themselves - but would be good to confirm

Reply:
@jen-machin - where are you tracking the specific bits for ADA to definitely review, are you doing that separately?